server/models/migrations/next/migrate.sql

```diff
@@ -13,3 +13,348 @@ INSERT INTO unit VALUES
 
 INSERT INTO `report` (`id`, `report_key`, `title_key`) VALUES
```

Comment:
@jeremielodi fix this error INSERT INTO `report` (`report_key`, `title_key`) VALUES